metagraph/src/annotation/representation/base/annotation.hpp

```diff
@@ -51,6 +51,9 @@ class MultiLabelAnnotation
 
     virtual void add_labels(const std::vector<Index> &indices,
                             const VLabels &labels) = 0;
+    virtual void add_label_counts(const std::vector<Index> &indices,
+                                  const VLabels &labels,
+                                  const std::vector<uint32_t> &counts);
```

Comment:
Is it significantly slower if you use `sdsl::int_vector<>` here with a width of 32 instead of `std::vector<uint32_t>`?

Reply:
I didn't try, but this doesn't make much sense, these vectors are limited in size, so there is no reason to pack them as much as possible.